examples/objective-c/auth_sample/AuthSample.xcodeproj/project.pbxproj

```diff
@@ -116,12 +116,15 @@
 			isa = PBXNativeTarget;
 			buildConfigurationList = 63E1E9A21B28CB2100EF0978 /* Build configuration list for PBXNativeTarget "AuthSample" */;
 			buildPhases = (
+				A0E971D0BB7E77C68F1B71CA /* 📦 Check Pods Manifest.lock */,
```

Comment:
You should upgrade to Cocoapods 1.0.1. The emojis were introduced in 1.0.0 and then removed (changed to `[CP]`). I wouldn't care about them except for that one time we saw Jenkins unhappy with them.